src/AppBundle/Committee/CommitteeManager.php

```diff
@@ -2,33 +2,73 @@
 
 namespace AppBundle\Committee;
 
+use AppBundle\Entity\Adherent;
 use AppBundle\Entity\Committee;
 use AppBundle\Entity\CommitteeMembership;
-use AppBundle\Repository\AdherentRepository;
-use AppBundle\Repository\CommitteeMembershipRepository;
+use AppBundle\Geocoder\Coordinates;
+use Doctrine\Common\Persistence\ManagerRegistry;
 
 class CommitteeManager
 {
-    private $adherentRepository;
-    private $membershipRepository;
-
-    public function __construct(
-        AdherentRepository $adherentRepository,
-        CommitteeMembershipRepository $membershipRepository
-    ) {
-        $this->adherentRepository = $adherentRepository;
-        $this->membershipRepository = $membershipRepository;
+    private const COMMITTEE_PROPOSALS_COUNT = 3;
+
+    private $registry;
+
+    public function __construct(ManagerRegistry $registry)
+    {
+        $this->registry = $registry;
     }
 
-    public function findCommitteeHostsList(Committee $committee): array
+    /**
+     * @param string $committeeUuid
+     *
+     * @return Adherent[] The hosts of the committee
+     */
+    public function getHosts(string $committeeUuid)
     {
-        $uuids = array_map(
-            function (CommitteeMembership $membership) {
-                return (string) $membership->getAdherentUuid();
-            },
-            $this->membershipRepository->findHostMemberships((string) $committee->getUuid())
-        );
-
-        return $this->adherentRepository->findList($uuids);
+        $hostMemberships = $this->registry->getRepository(CommitteeMembership::class)->findHosts($committeeUuid);
+
+        $hosts = array_map(function (CommitteeMembership $hostMembership) {
+            return $hostMembership->getAdherent();
+        }, $hostMemberships);
+
+        return $hosts;
+    }
+
+    public function findNearbyCommittees(Coordinates $coordinates, $limit = self::COMMITTEE_PROPOSALS_COUNT)
+    {
+        $data = [];
+        $committeeMembershipRepository = $this->registry->getRepository(CommitteeMembership::class);
+        $committees = $this->registry->getRepository(Committee::class)->findNearbyCommittees($limit, $coordinates);
+
+        foreach ($committees as $committee) {
```

Comment:
This section should go in the repository or in a `CommitteeCollection` specific class.